llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -24555,6 +24555,19 @@ SDValue AArch64TargetLowering::PerformDAGCombine(SDNode *N,
     if (auto R = foldOverflowCheck(N, DAG, /* IsAdd */ false))
       return R;
     return performFlagSettingCombine(N, DCI, AArch64ISD::SBC);
+  case AArch64ISD::BICi: {
+    KnownBits Known;
+    APInt DemandedBits =
+        APInt::getAllOnes(N->getValueType(0).getScalarSizeInBits());
```

Comment:
I would expect this not be all bits, but to only demand the bits that are not cleared by the BIC.  Again this might be difficult to test though, so it might be best to leave it as-is for the moment.